CONTRIBUTING.md

```diff
@@ -1,54 +1,59 @@
 # CONTRIBUTING
-
-Contributing to the official buyTogether Application
+Contributing to the official buyTogether application
 Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
-
 ---
-
-## Make changes locally
-
-- Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
-
-- Create a working branch 
-
-- Set up a development environment to begin working on the project all you need to do is 'vagrant up'
-
+## Prerequisites
+- Fork the repo to make your changes without affecting the original project until you're ready to merge.
+- Create a working branch.
+- Set up a development environment to begin working on the project, all you need to do is 'vagrant up'
 - Start with your changes!
-
 ---
-
-## Commit your update
-Commit the changes once you are happy with them. 
-
+## Commiting
+Please use the guide below inorder to follow our commit message convention:
+* [commit convention](https://cbea.ms/git-commit/#seven-rules)
 ---
-
 ## Pull Request
 - When you're finished with the changes, create a pull request, also known as a PR.
-- Please note that at least two team members of the group should review and approve the pull request before asking the mentors to review and merge.
-- The reviewer resolve the conversation to indicate that they are ok with the updated content.
+- If your PR depends on the merging of other PRs, indicate this on the commit message or on the comments.
```

Comment:
don't indicate it in the comment section, indicate it on the commit message. 
also indicate it as such: `depends on #<number of PR>` so it'll be linked.